gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java

```diff
@@ -178,6 +215,96 @@ public boolean accept(File file) {
   }
 
   /**
+   * Recursively load job configuration files under given URI of **directory of config files folder**
+   * todo
+   */
+  private static void loadGenericJobConfigsRecursive(List<Properties> jobConfigs, Properties rootProps,
+      Set<String> jobConfigFileExtensions, URI uri)
+      throws ConfigurationException, IOException {
+
+    /* create the generic file system */
+    org.apache.hadoop.conf.Configuration conf = new org.apache.hadoop.conf.Configuration();
+    FileSystem filesystem = FileSystem.get(uri, conf);
+
+    Path configDirPath = new Path(uri);
+    if (!filesystem.exists(configDirPath)) {
+      throw new RuntimeException("No specified .properties directory found :" + configDirPath.toString());
```

Comment:
There's a constant String named `JOB_PROPS_FILE_EXTENSION` with the value of "properties", that's why I put it so. 

Do we need to use ".pull" / ".job" ? I guess there's some historical problem which caused inconsistency.